packages/cli/src/constructs/tcp-monitor.ts

```diff
@@ -52,10 +69,29 @@ export interface TcpMonitorProps extends MonitorProps {
   request: TcpRequest
   /**
    * The response time in milliseconds where a check should be considered degraded.
+   * TCP checks have lower thresholds than HTTP checks due to protocol differences.
+   * 
+   * @defaultValue 4000
+   * @minimum 0
+   * @maximum 5000
+   * @example
+   * ```typescript
+   * degradedResponseTime: 1000  // Alert when TCP connection takes longer than 1 second
+   * ```
    */
   degradedResponseTime?: number
+  
   /**
    * The response time in milliseconds where a check should be considered failing.
+   * Maximum allowed value is lower for TCP checks compared to HTTP checks.
+   * 
+   * @defaultValue 5000
+   * @minimum 0
+   * @maximum 5000
```

Comment:
These are correct for the public API but it looks like the CLI endpoint does not enforce the same values. These are the effective values currently:

```
   * @defaultValue 20000
   * @minimum 0
   * @maximum 300000
```

This should probably be fixed in the backend.

Reply:
@sorccu Ok, that means we keep it this way for this PR, right?